app/webpacker/css/shared/utilities.scss

```diff
@@ -11,11 +11,11 @@
 .flex {
   display: flex;
 
-  &.align-center {
+  &.flex-align-center {
```

Comment:
Yep, but others rules exists with class `.align-center`: 

https://github.com/openfoodfoundation/openfoodnetwork/blob/7317d6ca34730af7b7dd0c9d66e6a8eab8766f4a/app/webpacker/css/admin/shared/typography.scss#L114

That's why I've added this prefix to avoid name collision.

Reply:
Oh, I see now, this will have the unintended effect of applying `text-align: center` to the flex items.